.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+### What's being changed:
+
+
+### Review checklist
+
+- [ ] Documentation has been updated, if necessary. Link to changed documentation:
```

Comment:
Here is In my opinion about documentation:
- Code should be well documented.
- Public API should have their own documentation tasks. Therefore, user documentation should be updated once Weaviate's alpha version is ready because APIs might change very often during development.